public_html/wp-content/plugins/pattern-directory/includes/admin-stats.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace WordPressdotorg\Pattern_Directory\Admin\Stats;
+
+use function WordPressdotorg\Pattern_Directory\Stats\{ get_meta_field_schema, get_snapshot_data, get_snapshots };
+use const WordPressdotorg\Pattern_Directory\Pattern_Post_Type\POST_TYPE as PATTERN_POST_TYPE;
+
+defined( 'WPINC' ) || die();
+
+/**
+ * Actions and filters.
+ */
+add_action( 'admin_menu', __NAMESPACE__ . '\add_subpage' );
+
+/**
+ * Register an admin page under Block Patterns for viewing stats.
+ *
+ * @return void
+ */
+function add_subpage() {
+	$parent_slug = add_query_arg( 'post_type', PATTERN_POST_TYPE, 'edit.php' );
+
+	$post_type_object = get_post_type_object( PATTERN_POST_TYPE );
+
+	add_submenu_page(
+		$parent_slug,
+		__( 'Pattern Stats', 'wporg-patterns' ),
+		__( 'Stats', 'wporg-patterns' ),
+		$post_type_object->cap->edit_posts,
+		PATTERN_POST_TYPE . '-stats',
+		__NAMESPACE__ . '\render_subpage'
+	);
+}
+
+/**
+ * Render the stats subpage.
+ *
+ * @return void
+ */
+function render_subpage() {
+	$schema         = get_meta_field_schema();
+	$data           = get_snapshot_data();
+	$snapshot_query = get_snapshots( array(), true );
+	$snapshots      = $snapshot_query->get_posts();
+
+	?>
+	<div class="wrap">
+		<h1 class="wp-heading-inline">
+			<?php esc_html_e( 'Pattern Stats', 'wporg-patterns' ); ?>
+		</h1>
+
+		<p>
+			This page is a work in progress. Someday there might be charts!
+		</p>
+
+		<h2>
+			Right now
+		</h2>
+
+		<table class="widefat striped">
+			<tbody>
+				<?php foreach ( $schema['properties'] as $field_name => $field_schema ) : ?>
+					<tr>
+						<td>
+							<abbr title="<?php echo esc_attr( $field_schema['description'] ); ?>">
+								<?php echo esc_html( $field_name ); ?>
+							</abbr>
+						</td>
```

Comment:
Rather than having the descriptions on hover, could they be added as a middle column? I'm on a large monitor so there is definitely enough space, but I don't think we need to worry much about smaller screens unless it's unusable…
<img width="1483" alt="Screen Shot 2021-07-16 at 4 08 46 PM" src="https://user-images.githubusercontent.com/541093/126003427-bd138b12-43e9-4768-a938-91476ee0cd00.png">

Reply:
b84b2c1